src/ctr/appengine.cc

```diff
@@ -0,0 +1,10 @@
+#include "appengine.h"
+
+
+namespace ctr {
+  AppEngine::Result AppEngine::fetch(const App& app) {
+    // TODO
+    return true;
+  }
+
+} // namespace ctr
```

Comment:
Is the "ctr" here implying the future code with be using the containerd "ctr" stuff? You tend to err on the side of overly verbose with names - this seems a bit terse to me

Reply:
>Is the "ctr" here implying the future code with be using the containerd "ctr" stuff?

No, I just meant a short name for `containers` and I've not planned to use any `ctr` stuff. Currently just some pieces of containerd's functionality is used by `composectl`.

Yes, I tend to be verbose when it comes to variables/classes/methods, not for namespaces though :).

I just couldn't come up with any reasonable and short name for this namespace, maybe just `compose`?

Reply:
It's preferable to name a sub-directory with the same name as a namespace. Maybe, calling it `composeapp` is a good name for both?

Reply:
lgtm

Reply:
86a0de0